app/src/main/java/edu/ucandroid/weathernotice/ui/main/RetrofitClientInstance.kt

```diff
@@ -8,9 +8,9 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
 object RetrofitClientInstance {
-    private var retrofit: Retrofit? = null;
+    private var retrofit: Retrofit? = null
 
-    private val BASE_URL = "https://api.weatherbit.io/"
+    private const val BASE_URL = "https://api.weatherbit.io/"
```

Comment:
Removed a semi-colon on line 11. On line 13, I made BASE_URL into a constant, because as stated earlier, we know the value of this at compile time, and it will not change, so it's in our interest to make sure that we tell the compiler this information. See the citation below for a discussion of why to use constants in your code. The discussion was on constants in C, but I would assume the general idea is the same.

Citations: [https://stackoverflow.com/questions/2953601/why-use-constants-in-programming](https://stackoverflow.com/questions/2953601/why-use-constants-in-programming)

Reply:
Yes... especially when you go back and forth between Java and Kotlin.  :)